app/src/main/java/org/mifos/mobile/ui/fragments/SavingAccountsTransactionFragment.kt

```diff
@@ -298,84 +299,70 @@ class SavingAccountsTransactionFragment : BaseFragment() {
     /**
      * Shows a filter dialog
      */
+     private lateinit var filterBinding: LayoutFilterDialogBinding
     private fun showFilterDialog() {
-        val inflater = activity?.layoutInflater
-        val dialogView = inflater?.inflate(R.layout.layout_filter_dialog, null, false)
-        val checkBoxPeriod: AppCompatCheckBox? = dialogView?.findViewById(R.id.cb_select)
-        val radioGroupFilter = dialogView?.findViewById<RadioGroup>(R.id.rg_date_filter)
-        tvStartDate = dialogView?.findViewById(R.id.tv_start_date)
-        tvEndDate = dialogView?.findViewById(R.id.tv_end_date)
+        filterBinding = LayoutFilterDialogBinding.inflate(layoutInflater)
+        val dialogView = filterBinding.root
+
+        val checkBoxPeriod: AppCompatCheckBox? = filterBinding.cbSelect
```

Comment:
do we need to explicitly declare AppCompatCheckBox ?

Reply:
Nope ig...should i remove it..?